src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.WorkItem.cs

```diff
@@ -167,12 +163,6 @@ public WorkItem ToProjectWorkItem(IAsyncToken asyncToken)
                         asyncToken);
                 }
 
-                public WorkItem With(ImmutableHashSet<IIncrementalAnalyzer> specificAnalyzers, IAsyncToken asyncToken)
```

Comment:
This method was unused, and didn't call `Union` at all, so I thought it best to remove before someone calls it and does who-knows-what